src/node/ext/call.cc

```diff
@@ -721,6 +721,14 @@ NAN_METHOD(Call::StartBatch) {
   }
   Local<Function> callback_func = info[1].As<Function>();
   Call *call = ObjectWrap::Unwrap<Call>(info.This());
+  if (call->wrapped_call == NULL) {
```

Comment:
I'm not sure about this but am wondering about one other thing:
if the `NULL` check of `call->wrapped_call` was atomic with the pending batches increment, then I think the `writeFailed` flag could be gotten rid of